The cached data values were being collected when an OP_EXEC call was made.
This is incorrect as the values are only for logging purposes.  It's believed
this caused an occasional crash in certain instances.

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

diff --git a/pseudo.c b/pseudo.c
index efee436..62f28f3 100644
--- a/pseudo.c
+++ b/pseudo.c
@@ -1021,7 +1021,8 @@ pseudo_op(pseudo_msg_t *msg, const char *program, const char *tag) {
 	if (msg->op != OP_MAY_UNLINK &&
 	    msg->op != OP_DID_UNLINK &&
 	    msg->op != OP_CANCEL_UNLINK &&
-	    msg->op != OP_UNLINK) {
+	    msg->op != OP_UNLINK &&
+	    msg->op != OP_EXEC ) {
 		cache_msg = *msg;
 
 		if (path_by_ino) {
